cmd/demo/node.go

```diff
@@ -303,7 +350,24 @@ func findConfig(id wallet.Address) (string, *netConfigEntry) {
 	return "", nil
 }
 
-func (n *node) Handle(req *client.ChannelProposal, res *client.ProposalResponder) {
+func (n *node) HandleUpdate(update client.ChannelUpdate, resp *client.UpdateResponder) {
+	n.mtx.Lock()
+	defer n.mtx.Unlock()
+	n.log.WithField("ID", update.State.ID).Trace("Channel update")
```

Comment:
Since you use this logger twice, locally define and reuse it as `log :=  n.log.WithField("channel", update.State.ID)`.